src/components/Frame/README.md

```diff
@@ -646,17 +644,16 @@ function FrameExample() {
     </Modal>
   );
 
-  const theme = {
-    frameOffset: '60px',
-    logo: {
-      width: 124,
-      topBarSource:
-        'https://cdn.shopify.com/s/files/1/0446/6937/files/jaded-pixel-logo-color.svg?6215648040070010999',
-      contextualSaveBarSource:
-        'https://cdn.shopify.com/s/files/1/0446/6937/files/jaded-pixel-logo-gray.svg?6215648040070010999',
-      url: 'http://jadedpixel.com',
-      accessibilityLabel: 'Jaded Pixel',
-    },
+  const theme = {frameOffset: '60px'};
```

Comment:
**question (non-blocking):** Are we keeping this `theme` const around as a "TODO" for the follow up issue to add back frame offset support?